cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2203,6 +2206,42 @@ if(ARROW_WITH_RAPIDJSON)
   endif()
 endif()
 
+macro(build_qpl)
```

Comment:
Is it possible to add the dependency to `cpp/thirdparty/versions.txt`? It will be a lot easier for developers if offline build is supported.

Reply:
Yes, i'll update it.